cogeo_mosaic/backends/dynamodb.py

```diff
@@ -165,7 +166,9 @@ def _write_items(self, items: List[Dict]):
                 for item in progitems:
                     batch.put_item(item)
 
-    @functools.lru_cache(maxsize=512)
+    @cached(
+        TTLCache(maxsize=512, ttl=300), key=lambda self: hashkey(self.path),
+    )
```

Comment:
simple cache, based only on the table + region name

Reply:
Is `maxsize` in bytes or items?

Reply:
Good question, I thought it was in bytes, but it's in items! which is still ok I guess 💁

Reply:
> All these decorators wrap a function with a memoizing callable that saves up to the maxsize most recent calls, u